generation/new_client/README.md

```diff
@@ -143,11 +143,11 @@ Example: `alloydb`
 
 ### Proto path
 
-The script takes "proto path" parameter. This is path from google3's root (`google3/third_party/googleapis/stable/`) to the
+The script takes "proto path" parameter. This is path from `googleapis/googleapis/blob/master`' root to the
```

Comment:
Can you clarify what is googleapis/googleapis/blob/master?

Reply:
Yep - it's the Github mirror of `google3/third_party/googleapis/stable`. I updated since we are converting this doc to use all external references.